style.css

```diff
@@ -404,9 +416,185 @@ form button:active {
   border-radius: 25px;
 }
 
-form input:hover {
-  text-decoration: underline;
-  text-decoration-color: #18a0fa;
-  border: 3px solid #18a0fa;
-  text-decoration-thickness: 5px;
+footer .social-media {
+  display: none;
+}
+
+@media only screen and (min-width: 992px) {
+  body {
+    width: 97vw;
+    overflow-x: hidden;
+  }
+
+  .first-page-section {
+    background-repeat: repeat;
+    height: 1000px;
+  }
+
+  .welcome {
+    display: none;
+  }
+
+  .menu-button {
+    display: none;
+  }
+
+  .header {
+    display: flex;
+    justify-content: center;
+  }
+
+  footer .social-media {
+    margin-top: 200px;
+    display: flex;
+    border-top: 1px solid #dfe1e6;
+  }
+
+  .contact-form .social-media,
+  .contact-form a {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: flex;
+    justify-content: center;
+    gap: 65px;
+  }
+
+  .desktop-nav li a:link {
+    text-decoration: none;
+    cursor: pointer;
+    color: #344563;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .desktop-nav li a:visited {
+    color: #344563;
+  }
+
+  .parag {
+    width: 50vw;
+  }
+
+  .indicator-div {
+    display: none;
+  }
+
+  .projects {
+    display: grid;
+    justify-content: space-evenly;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    grid-gap: 50px 20px;
+    margin-top: 50px;
+  }
+
+  .project {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-bottom: 20px;
+    border: 0.5px solid rgba(124, 124, 124, 0.356);
+    border-radius: 8px;
+  }
+
+  .project-img {
+    width: 100%;
+  }
+
+  .img {
+    height: auto;
```

Comment:
- [ ] Kindly for every project image use the height value same as the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A2068)template.

Reply:
I had changed the height of every portfolio project and the height of the placeholder images as they are in the Figma template.